app/views/webhooks/_detail.html.slim

```diff
@@ -21,6 +21,6 @@
         th Password
         td
           - if webhook.password.present?
-            = webhook.password
+            = '*' * webhook.password.length
```

Comment:
I doesn't really matter if it's a random or fixed length. The fixed length string however is really easy to implement.